app/assets/stylesheets/govuk_publishing_components/components/govspeak/_typography.scss

```diff
@@ -33,32 +33,29 @@
   // Block quotes
 
   blockquote {
-    padding: 0 0 0 govuk-spacing(4);
+    padding: 0 0 0 govuk-spacing(6);
     margin: 0;
     border: 0;
 
-    @include govuk-media-query($from: desktop) {
-      margin: 0 0 0 (- govuk-spacing(6));
-    }
-
     p {
-      padding-left: govuk-spacing(3);
-
-      @include govuk-media-query($from: tablet) {
-        padding-left: govuk-spacing(6);
-      }
+      position: relative;
 
-      &::before {
+      &:first-of-type::before {
         content: "\201C";
-        float: left;
-        clear: both;
-        margin-left: (- govuk-spacing(3));
+        position: absolute;
+        top: 0;
+        right: 100%;
       }
 
-      &:last-child::after {
+      &:last-of-type::after {
         content: "\201D";
       }
     }
+
+    cite {
+      display: block;
+      margin-bottom: govuk-spacing(4);
```

Comment:
Would there by any mileage in amending the margin ruleset for the blockquote as a whole (currently on line 37) to something like `margin: 0 0 govuk-spacing(4) 0`? Only because I recall a conversation we had a while back about margins and, if I remember correctly, one of the things we mused about was moving all (or the majority) of spacing to the parent component to prevent child elements from causing confusion when looking at spacing.

This all looked good though so totally happy to leave things as they are if that's preferable.

Reply:
That's a good idea. I'm not going to add additional rules for the paragraphs to remove their margin, because their margin is the same as the blockquote they just overlap. But this also means we don't need styles for the `cite` anymore, so this feels a bit cleaner and more efficient.